lib/netdev-rte-offloads.c

```diff
@@ -1471,3 +1483,51 @@ netdev_dpdk_flow_del(struct netdev *netdev OVS_UNUSED, const ovs_u128 *ufid,
     return -1;
 }
 
+
+
+static void netdev_rte_port_preprocess(struct netdev_rte_port * rte_port,
+                                       struct dp_packet *packet)
+{
+    switch (rte_port->rte_port_type) {
+        case RTE_PORT_TYPE_VXLAN:
+            {
+                // VXLAN table failed to match on HW. we do however
+                // no the port-id so we just pop it here.
+                if (rte_port->netdev->netdev_class->pop_header) { 
+                    rte_port->netdev->netdev_class->pop_header(packet);
+                    packet->md.in_port.odp_port = rte_port->port_no;
+                    reset_dp_packet_checksum_ol_flags(packet);
+                }
+            }
+            break;
+        case RTE_PORT_TYPE_NONE:
+        case RTE_PORT_TYPE_DPDK:
+            break;
+
+    }
+    return;
+}
+
+
+/**
+ * @brief - we got a packet with special mark, means we need to run 
```

Comment:
dont you want to mention that special mark is received when we get a miss on a table and de-facto this is an exception handling?